spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-dead-letter-sample/src/main/java/com/example/SourceExample.java

```diff
@@ -35,7 +34,11 @@ public class SourceExample {
 
   private final Logger log = LoggerFactory.getLogger(this.getClass());
 
-  @Autowired private Sinks.Many<UserMessage> postOffice;
+  private final Sinks.Many<UserMessage> postOffice;
```

Comment:
No, this is a code refactor since [field injection is not recommended](https://stackoverflow.com/questions/39890849/what-exactly-is-field-injection-and-how-to-avoid-it).